src/app/page.js

```diff
@@ -1,95 +1,94 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+'use client'
+import { useRef } from "react";
+import Popup from 'reactjs-popup';
```

Comment:
This lib is not included in the package.json, hence it fails to build if I checkout your branch on my machine